README.md

```diff
@@ -11,10 +11,12 @@
 1. To set-up the environment for you'll need:
 
 - [Ruby](https://www.ruby-lang.org/en/downloads/) (2.4 or above)
+- [Python 2](https://www.python.org/downloads/)
 - [Jekyll](https://jekyllrb.com/)
-- [NodeJS](https://nodejs.org/en/) (0.12 or greater)
+- [NodeJS](https://nodejs.org/en/) (0.10)
```

Comment:
@sparkica: I can't for the life of me get `gulp build` to run successfully on Node 0.12 and MacOS. Downgrading to Node 0.10 makes it work. Is there something that depends on Node 0.12 specifically?

Reply:
```
dragonpy.com (nix)$ gulp build
[06:33:36] Using gulpfile ~/work/dragonpy.com/gulpfile.js
[06:33:36] Starting 'build'...
[06:33:36] Starting 'sass'...
[06:33:36] 'sass' errored after 67 ms
[06:33:36] Error: Missing binding /Users/zupo/work/dragonpy.com/node_modules/node-sass/vendor/darwin-x64-64/binding.node
Node Sass could not find a binding for your current environment: OS X 64-bit with Node.js 10.x
Found bindings for the following environments:
  - OS X 64-bit with Node.js 12.x
This usually happens because your environment has changed since running `npm install`.
Run `npm rebuild node-sass` to download the binding for your current environment.
    at module.exports (/Users/zupo/work/dragonpy.com/node_modules/node-sass/lib/binding.js:15:13)
    at Object.<anonymous> (/Users/zupo/work/dragonpy.com/node_modules/node-sass/lib/index.js:14:35)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/Users/zupo/work/dragonpy.com/node_modules/gulp-sass/index.js:162:21)
[06:33:36] 'build' errored after 70 ms
```

Reply:
`npm rebuild node-sass` does not help